src/portable/config.rs

```diff
@@ -24,6 +24,8 @@ pub struct SrcConfig {
 pub struct SrcEdgedb {
     #[serde(default)]
     pub server_version: Option<toml::Spanned<Query>>,
+    #[serde(default)]
+    pub schema_directory: Option<toml::Spanned<String>>,
```

Comment:
I think we want `schema_dir`. Since we already consistently use that contraction in command-line (`--schema-dir` and `--project-dir`). I don't think remembering when to abbreviate and when not is a good idea.